go.mod

```diff
@@ -118,3 +118,5 @@ require (
 	google.golang.org/protobuf v1.32.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace github.com/equinix/equinix-go-sdk => ../../equinix-sdk-go/
```

Comment:
Is this 0.34.0?

Reply:
This points to some manually patched open API specs while I wait for my spec PR to land.

I pushed this up to my fork here: https://github.com/0xch4z/equinix-sdk-go/commit/82dbfb149c9f3940e06eaff6c258cfaf407deeaa

Reply:
AFAIK, the `Facility` fields in that spec should be marked as deprecated.

Reply:
fixed now